include/kernels/dslash_twisted_mass_preconditioned.cuh

```diff
@@ -58,7 +58,7 @@ namespace quda
     typedef Matrix<complex<real>, nColor> Link;
     const int their_spinor_parity = nParity == 2 ? 1 - parity : 0;
 
-#pragma unroll
+#pragma unroll nDim
     for (int d = 0; d < nDim; d++) { // loop over dimension
```

Comment:
Is there a reason this is a loop over `nDim` and not over `4` like for the other fermion types? (Not asking this to be changed if it doesn't really matter, but it's a noteworthy difference.)

Reply:
It's a case that it was `nDim` elsewhere, but I ended up hardcoding to 4 where needed.  E.g., in the dslash_wilson.cuh kernel is used for 5-d and 4-d fermion kernels, so I hardcoded to 4.  For this operator, the number of grid dimensions is equal to stencil dimension, so might as well leave the code generic.

Older code is likely hard-coded to 4 where it could be more generic.